main/Entities/Intern.java

```diff
@@ -11,7 +11,7 @@
     */
     private final String internName;
     private final int internAge;
-    private final HashMap<String, Integer> internSkills;
+    private final HashMap<String, Double> internSkills;
```

Comment:
The internSkills should be a hash map with String, Integer. I think merge from main to solve this problem..?

Reply:
It was initially Integer, but there were some tests that used a double, and also the textfiles for internSkills contained double values, so just to be safe I changed that to double

Reply:
okok! double it is